app/pages/Profile/sections.css

```diff
@@ -1,83 +1,160 @@
 
 .profile > .section-container {
-	background-color: white;
-
-	& .section {
-	  background-color: white;
-	  padding-left: 56px;
-	  padding-right: 56px;
-	  padding-bottom: 48px;
-
-	  & h4 {
-	  	border-top: 1px solid #ddd;
-		display: block !important;
-		font-size: 28px !important;
-		margin: 16px !important;
-		padding-top:40px;
-		width: 100% !important;
+  background-color: white;
+  display: block;
+  float: left;
+  width:100%;
+
+  & .section {
+	  border-bottom: 1px solid #ddd;
+		background-color: white;
+		margin: 0px 72px 64px;
+		padding: 0 0 16px;
+
+		&:last-child {
+			border-bottom: none;
+		}
+
+		& .section {
+			border: none;
+			margin: 0px;
+			padding: 0px;
+		}
+
+		& h4 {
+			display: block;
+			font-size: 28px;
+			margin: 16px;
+			padding-top: 40px;
+			width: 100%;
 	  }
 
-	  & .columns {
+		& .columns {
+			width: auto;
+
+			& .section-title {
+				margin: 0 0 16px 0;
+				width: 300px;
+			}
+
+			& .viz {
+				width: auto;
+
+				&:first-child {
+					margin-right: 16px;
+				}
+			}
+		}
+
+		& .rows {
+
+			& .viz {
+
+				&:first-child {
+					margin-right: 0px;
+				}
+			}
+		}
+
+		& .margin-right {
+
+			& .viz {
 
-	  	& .section-text {
-		  font-size: 16px;
+				&:first-child {
+					margin-right: 16px;
+				}
+			}
+		}
+
+		& .section-title {
+		  display: block;
+		  font-size: 24px;
+		  margin: 16px 16px 16px 0px;
+		  padding-top: 0px;
+		  width: 500px;
+		}		
+	
+		& .dropdown {
```

Comment:
I moved the dropdown CSS to `components/Selector.css` so that both the Profile and Map could share the styles. Can you reduce the specific styles here to only include the *exceptions* to the shared styles?